code/modules/unit_tests/say.dm

```diff
@@ -110,45 +110,33 @@
 
 	// speaking
 	speaker.say("The quick brown fox jumps over the lazy dog")
-	TEST_ASSERT(handle_speech_result, "Handle speech signal was not fired")
-	TEST_ASSERT(handle_hearing_result, "Handle hearing signal was not fired")
-	handle_speech_result = FALSE
-	handle_hearing_result = FALSE
+	TEST_ASSERT(handle_speech_result == 1, "Handle speech signal was not fired")
```

Comment:
TEST_ASSERT_EQUAL for this and everything else

Reply:
https://github.com/tgstation/tgstation/blob/ad3aea8f6f2778d8c1fabd8cf95bfcb0f56c5bda/code/modules/unit_tests/say.dm#L108-L124